docs/introduction/building-and-running.md

```diff
@@ -0,0 +1,43 @@
+# Building and Running a Project
+
+The `subo` command line tool is used again here to build and run your Atmo project.
+
+## Building
+
+Inside the `important-api` directory run: 
+
+```
+subo build .
+```
+
+This automatically compiles each of your Runnables in a Docker container and bundles them together in `runnables.wasm.zip` to be used in Atmo. 
+
+```
+⏩ START: building runnables in .
+ℹ️  🐳 using Docker toolchain
+⏩ START: building runnable: helloworld (rust)
+    Updating crates.io index
+[...]
+
+✅ DONE: bundle was created -> runnables.wasm.zip @ v0.1.0
+```
+
+{% hint style="info" %}
+If you prefer not to use Docker, you can also [build your Runnables locally](https://github.com/suborbital/subo/blob/main/docs/get-started.md#building-without-docker).
```

Comment:
I would use the word 'natively' instead of 'locally', because that implies to me that the Docker toolchain is somehow remote :)